web/packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -97,6 +97,29 @@ function TdpClientCanvas(props: Props) {
     }
   }, [client, clientOnPngFrame]);
 
+  useEffect(() => {
+    if (client) {
+      const canvas = canvasRef.current;
+      const updatePointer = (pointer: {data: ImageData, hotspot_x: number, hotspot_y: number}) => {
+        if (pointer.data === undefined) {
+          canvas.style.cursor = "none";
+          return;
+        }
+        const cursor = canvas.ownerDocument.createElement("canvas");
```

Comment:
Is there a reason this doesn't work?
```suggestion
        const cursor = document.createElement('canvas');
```

Reply:
Only my lack of knowledge about TypeScript